src/sql/base/browser/ui/table/media/slickColorTheme.css

```diff
@@ -148,7 +148,7 @@
 
 .vs ::-webkit-scrollbar {
 	width: 14px;
-	height: 10px;
+	height: 12px;
```

Comment:
This will affect the scrollbar everywhere. I'm not sure why we are even defining this in this css file.

Reply:
I think this is fine.  The vertical scrollbars are too small and has been noted in several bugs where people have complained about having difficulty using it.